etc/completions

```diff
@@ -1,5 +1,807 @@
-$command $opt* ($revision|$path|$remote)* --? $path+
+(add|stage) $opt* --? $path+
```

Comment:
Good call. It would probably be helpful to not only have a section header comment (like I do below), but also reference the manual page where users can find out about the file format.